app/main.js

```diff
@@ -1,4 +1,5 @@
-const { app, BrowserWindow, ipcMain, shell } = require('electron');
+const electron = require('electron');
```

Comment:
I dont really see why its necessary to declare electron separately here. The only use seems to be the shell.openExternal function. shell was already imported via object destruction.